projects/chaos/inc/relay_fsm.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <stdint.h>
+
+#include "can_ack.h"
+#include "fsm.h"
+#include "relay_id.h"
+#include "status.h"
+
+#define RELAY_FSM_MAX_RETRIES 2
+#define NUM_RELAY_FSMS NUM_RELAY_IDS
+
+// Initializes the relay_fsm module.
+void relay_fsm_init(void);
+
+// Creates a new relay FSM instance for a RelayId.
+StatusCode relay_fsm_create(FSM* fsm, RelayId relay_id, const char* fsm_name,
+                            uint32_t ack_device_bitset);
+
+// Sets |e| with the event to open the relay corresponding to |relay_id|.
+StatusCode relay_fsm_open_event(RelayId relay_id, Event* e);
```

Comment:
Maybe "populates" instead of sets? Also watch out for the spacing of your `*`'s (I see another case in `relay_fsm_create`. I thought we had a format/lint rule for that?

Reply:
Done. I'll look into the `.clang-fomrat` file in a separate PR.